### PR TITLE
Raster extensions reshuffle

### PR DESCRIPTION
**Pull request by emmanuelmathot**
https://github.com/stac-extensions/raster/issues/10

### CODE REVIEW

**Comment by emmanuelmathot** [comment]
Waiting for https://github.com/stac-extensions/processing/pull/2 to link processing

File: README.md [hunk 1]
```diff
@@ -30,89 +30,65 @@ more raster assets (RGB combination, simple band value processing) and to create
...
+| sampling            | string    | One of `area` or `point`. Indicates whether a pixel value should be assumed to represent a sampling over the region of the pixel or a point sample at the center of the pixel.   |
+| data_type           | string    | The data type of the band. One of the [data types as described above](#data-types).                                                                                              |
+| bits_per_sample     | number    | The actual number of bits used for this band. Normally only present when the number of bits is non-standard for the `datatype`, such as when a 1 bit TIFF is represented as byte |
+| spatial_resolution          | number  | Average spatial resolution (in meters) of the pixels in the band.                                                                                                                                                           |
+| statistics          |  \[[Statistics Object](#statistics-object)]    | Statistics of all the pixels in the band                                                                                                                                         |
```
Comment (lossyrob) [inline README.md:46]: Remove `[]` as it's not a list

File: README.md [hunk 2]
```diff
@@ -30,89 +30,65 @@ more raster assets (RGB combination, simple band value processing) and to create
...
+- `cfloat32`: 32-bit complex float
+- `cfloat64`: 64-bit complex float
+- `other`: Other data type than the ones listed above (e.g. boolean, string, higher precision numbers)
+
+### Statistics Object
+
+| Field Name          | Type      | Description                                                                                                                                                                      |
+| ------------------- | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| mean          | number    | mean value of all the pixels in the band                                                                                                                                         |
+| min           | number    | minimum value of the pixels in the band                                                                                                                                          |
```
Comment (lossyrob) [inline README.md:88]: Change to `minimum` and `maximum`

File: README.md [hunk 3]
```diff
@@ -158,46 +138,32 @@ and the 0 limnimetric scale to compute a water level.
...
-| bands             | \[[string](#band-selector)] | An ordered array of string following [Band Selector](#band-selector) syntax. If given, requires at least one band. **REQUIRED** if no `band_expression` specified                                                          |
-| band_expression   | string                      | Band math expression (e.g `(b4-b1)/(b4+b1)`). **REQUIRED** if no `bands` specified                                                                                                                                         |
-| resampling_method | string                      | Resampling method, one of `nearest`, `average`, `bilinear` or `cubic`.                                                                                                                                                     |
-| color_map         | string                      | Identifier of a color mapping. Currently based on [rio-tiler color maps](https://cogeotiff.github.io/rio-tiler/colormap/) that includes some from Matplotlib and some custom ones that are commonly used with raster data. |
-
-## Band Selector
+## Raster Composition using `virtual:assets`
```
Comment (lossyrob) [inline README.md:152]: Creating a new 'composite' extension that will combine virtual:assets and 'raster'

**Review by lossyrob — COMMENTED** [review]
Adding some notes from a review call between myself, Matthais Emmanuel, and Matt

**Pull request merged**